python/ray/tune/suggest/bayesopt.py

```
@@ -61,10 +64,11 @@ class BayesOptSearch(Searcher):
 
     def __init__(self,
                  space,
-                 metric="episode_reward_mean",
+                 metric,
```

can we leave this in? this may break existing codes

---

I thought that since there was no true default metric it was misleading to live that in. Am I mistaken?

---

hmm, I guess that's a good point